src/pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -517,7 +515,7 @@ def _set_parallel_devices_and_init_accelerator(self) -> None:
         self._gpus = self._devices_flag if not self._gpus else self._gpus
         self._tpu_cores = self._devices_flag if not self._tpu_cores else self._tpu_cores
 
-        self._set_devices_flag_if_auto_select_gpus_passed()
+        self._set_devices_flag_if_count_passed()
```

Comment:
We could merge this with `Accelerator.parse_devices` (see line below). I.e., we would implement `GPUAccelerator.parse_devices` by calling the `pick_multiple_gpus` function there.